<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phy6 Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>580</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Справочник физических величин</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Справочник формул</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Научнный калькулятор</string>
     </attribute>
     <widget class="QLCDNumber" name="table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>80</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>20</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>100</width>
        <height>440</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eq">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>100</y>
        <width>100</width>
        <height>440</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_sin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>170</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_plus">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pow">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>460</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dot">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>170</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>170</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_minus">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mult">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_div">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>370</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_sqrt">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>460</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>√</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pi">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>370</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>π</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fact">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>460</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_op">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>460</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>±</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn0">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_journal">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Журнал</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>190</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Таблицы</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Константы</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Планировщик задач</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_cases">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>370</width>
        <height>530</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>370</width>
        <height>430</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Время следующего события:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Описание события:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add_case">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить событие</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete_case">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>510</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить событие</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_case">
      <property name="geometry">
       <rect>
        <x>169</x>
        <y>480</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
